### ISSUE TITLE
Print more activity info

### ISSUE THREAD

**Issue by bajtos** [filecoin-station/desktop#1123]
At the moment, the activity log typically shows a bunch of fail and restart messages, which looks a bit strange after a while. But it's pretty much the only log Station produces now. If we remove it, then the activity log will become empty. 

Also: People are getting confused why their Stations do more or less jobs than other people, adding more data sounds useful.

Let's add more activities, for example:

- When SPARK schedules rewards for the previous round, we can log an activity, something along the lines of “0.0025 FIL was added to your scheduled rewards”
- Whenever Station completes 100 jobs, it can log another activity - “You completed another 100 jobs. Well done!”
- SPARK can measure retrieval bandwidth consumed and log an activity every X MB, e.g “You contributed 100MB of your network bandwidth to improve Filecoin retrievals. Thank you!”
- periodic updates (which could eventually ben numbers in the header) like "x retrievals tested in the last 10 minutes"

**Comment by patrickwoodhead** [comment]
I think if we get an error, we should move it from the logs into a banner or modal, and prompt the operator to check their connection and/or hard refresh station

**Comment by juliangruber** [comment]
For any error? Station wants to stay in the background at all times - if there's a way it can recover by simply retrying the operation, I don't think there should be any user interaction. So far we are not aware of errors that users can fix themselves, or are there some?

**Comment by bajtos** [comment]
>  Station wants to stay in the background at all times - if there's a way it can recover by simply retrying the operation, I don't think there should be any user interaction.

+1

**Comment by patrickwoodhead** [comment]
> For any error? Station wants to stay in the background at all times - if there's a way it can recover by simply retrying the operation, I don't think there should be any user interaction. So far we are not aware of errors that users can fix themselves, or are there some?

Yeah - agreed we should definitely retry in the background to see if we can get things going again. I'm thinking if there are any other edge cases which might require a hard refresh or a restart initiated by the user. If no such edge cases exists, then let's do it all automatically behind the scenes.
